src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -316,6 +316,11 @@ public StorageClientFactory StorageClientFactory
             }
         }
 
+        /// <summary>
+        /// If set, apply this timeout attribute to all functions. 
```

Comment:
"Default timeout configuration to apply to all functions. If TimeoutAttribute is explicitly applied to a function, the values specified by the attribute will take precedence"